### PR TITLE
Feature newpreformancedata

### PR DESCRIPTION
**Pull request by GlenHani**

### CODE REVIEW

File: INZFS.MVC/Models/DynamicForm/MultiSelectInputModel.cs [hunk 1]
```diff
@@ -17,9 +17,9 @@ public class MultiSelectInputModel : BaseModel, IValidatableObject
 
         protected override IEnumerable<ValidationResult> ExtendedValidation(ValidationContext validationContext)
         {
-            if (Mandatory == true  && MarkAsComplete)
+            if (Mandatory == true && MarkAsComplete)
```
Comment (lorenzoAtBEIS) [inline INZFS.MVC/Models/DynamicForm/MultiSelectInputModel.cs:20]: With && MarkAsComplete causes a null reference error when no option is selected. Remove or develop a new solution to handle null references.

File: INZFS.MVC/Models/DynamicForm/YesornoInputModel.cs [hunk 2]
```diff
@@ -13,7 +13,7 @@ public class YesornoInputModel : BaseModel, IValidatableObject
     {
         protected override IEnumerable<ValidationResult> ExtendedValidation(ValidationContext validationContext)
         {
-            if (Mandatory == true)
+            if (Mandatory == true && MarkAsComplete)
```
Comment (lorenzoAtBEIS) [inline INZFS.MVC/Models/DynamicForm/YesornoInputModel.cs:16]: With && MarkAsComplete causes a null reference error when no option is selected. Remove or develop a new solution to handle null references.

**Review by lorenzoAtBEIS — CHANGES_REQUESTED** [review]
Null reference errors when no option selected on radio inputs

**Review by RSA99 — DISMISSED** [review]
Reviewed & Approved

File: INZFS.MVC/Models/DynamicForm/MultiSelectInputModel.cs (hunk 1 above)
Comment (GlenHani) [reply INZFS.MVC/Models/DynamicForm/MultiSelectInputModel.cs:20]: Null Error Fixed with try catch

File: INZFS.MVC/Models/DynamicForm/YesornoInputModel.cs (hunk 2 above)
Comment (GlenHani) [reply INZFS.MVC/Models/DynamicForm/YesornoInputModel.cs:16]: fixed

**Comment by GlenHani** [comment]
> Null reference errors when no option selected on radio inputs

fixed

**Comment by GlenHani** [comment]
OLD

**Pull request closed without merging**